JCDispatch/build.gradle

```diff
@@ -0,0 +1,76 @@
+plugins {
+    id "application"
+    id "java"
+    id "eclipse"
+    id "idea"
+    id "jacoco"
+}
+
+apply from: "https://raw.githubusercontent.com/kryptnostic/gradles/master/mavenRepos.gradle"
```

Comment:
This should reference the new `repos.gradle` project in `github.com/openlattice/gradles`.